src/v/wasm/tests/wasm_cache_test.cc

```diff
@@ -207,16 +209,16 @@ TEST_F(WasmCacheTest, CachesEngines) {
     auto factory = ss::make_foreign(make_factory(meta));
     static thread_local ss::shared_ptr<engine> live_engine;
     invoke_on_all([&factory] {
-        auto engine_one = factory->make_engine();
-        auto engine_two = factory->make_engine();
+        auto engine_one = factory->make_engine(nullptr);
```

Comment:
nit: can we make a dummy logger for this? It's the downside of this pattern, but I think it's better overall.

Reply:
done